### PR TITLE
Feat: Setup our own backend Cart. No more cartql

### PR DESCRIPTION
**Pull request by sareacct91**

### CODE REVIEW

File: server/model/User.js [hunk 1]
```diff
@@ -63,9 +66,18 @@ userSchema.pre('save', async function(next) {
   if (this.isNew || this.isModified('password')) {
     this.password = await bcrypt.hash(this.password, 10);
   }
+  if (this.isNew) {
+    await model('cart').create({id: this._id});
+  }
   next();
```
Comment (sareacct91) [inline server/model/User.js:72]: Create/Delete cart when user is create/delete

File: server/routes/index.js
Comment (sareacct91) [inline server/routes/index.js:1]: Setup API endpoint to quickly seed, create/delete a test user, without stopping the server

File: server/schemas/resolvers.js
Comment (sareacct91) [inline server/schemas/resolvers.js:1]: Change the cart query/mutations to use our own Card model

File: server/utils/cartOps/shapeCartData.js
Comment (sareacct91) [inline server/utils/cartOps/shapeCartData.js:1]: Shape the Cart data to match the old interface to the client

**Review by DaniDrury — APPROVED** [review]
Nice work!

**Pull request merged**